src/controllers/partyController.js

```diff
@@ -0,0 +1,21 @@
+import party from '../db/partyDb';
+
+class PartyController {
+  static createParty(req, res) {
+    const newParty = {
+      id: party.length + 1,
+      name: req.body.name,
+      hqaddress: req.body.hqaddress,
```

Comment:
`hqaddress` should be in `camselCase` `hqAddress`

Reply:
aiit tnx i will adjust it